Source/WebCore/platform/mediastream/mac/BaseAudioSharedUnit.cpp

```diff
@@ -52,6 +54,8 @@ void BaseAudioSharedUnit::addClient(CoreAudioCaptureSource& client)
 
 void BaseAudioSharedUnit::removeClient(CoreAudioCaptureSource& client)
 {
+    RELEASE_LOG_ERROR(WebRTC, "BaseAudioSharedUnit::removeClient %p", this);
```

Comment:
Ditto

Reply:
This should be removed, I did it in the new patch